### ISSUE TITLE
Transfer Learning

### ISSUE THREAD

**Issue by jasony93** [zzxvictor/License-super-resolution#6]
I tried to implement transfer learning with the previously trained checkpoint, however it does not seem plausible.
What I did was simply, load checkpoint and trained my generator w/o discriminator.
Below is the code for loading checkpoint.

generator = RRDBNet(blockNum=10)
optimizer = tfk.optimizers.Adam(learning_rate=1e-3)
generator.compile(loss=contentLoss, optimizer=optimizer, metrics=[psnr, ssim])

generator_checkpoint = "checkpoint"
generator.load_weights(generator_checkpoint)

history = generator.fit(
    x=trainData,
    validation_data=valData,
    epochs=12,
    verbose=1,
    steps_per_epoch=len(train) // batch_size,
    validation_steps=len(val) // batch_size
)

Training Does start, however psnr diverges and becomes nan within few training steps.
